src/main/resources/com/google/api/codegen/java/main.snip

```diff
@@ -274,7 +294,18 @@
     @case "RequestObjectMethod"
       {@requestObjectMethod(apiMethod)}
     @case "CallableMethod"
-      {@callableMethod(apiMethod)}
+    # TODO create a GrpcStreaming ApiMethodType
```

Comment:
Nit: Is that really a nit?

I view this as a more "opportunistic" TODO - if someone is modifying this code, then take care of it; otherwise, I don't know that it's important enough for an issue to be filed.